core/src/main/java/org/geysermc/geyser/registry/mappings/versions/MappingsReader_v1.java

```diff
@@ -404,9 +405,7 @@ private CustomBlockComponentsMapping createCustomBlockComponentsMapping(JsonNode
                     JsonNode boneVisibility = geometry.get("bone_visibility");
                     if (boneVisibility.isObject()) {
                         Map<String, String> boneVisibilityMap = new Object2ObjectOpenHashMap<>();
-                        boneVisibility.fields().forEachRemaining(entry -> {
-                            boneVisibilityMap.put(entry.getKey(), entry.getValue().isBoolean() ? (entry.getValue().asBoolean() ? "1" : "0") : entry.getValue().asText());
-                        });
+                        boneVisibility.fields().forEachRemaining(entry -> boneVisibilityMap.put(entry.getKey(), entry.getValue().isBoolean() ? (entry.getValue().asBoolean() ? "1" : "0") : entry.getValue().asText()));
```

Comment:
Can we keep the old format and pull the key and value out to variables for readability